app/controllers/forms/archived_controller.rb

```diff
@@ -0,0 +1,9 @@
+module Forms
+  class ArchivedController < ApplicationController
+    after_action :verify_authorized
+    def show_form
```

Comment:
Missing a newline before the def

Reply:
Is this something we can/should make rubocop do?

Reply:
You'd think so, yes.

Reply:
Added the blank line, bit annoying that rubocop isn't catching this